src/status_im/ui/components/connectivity/subs.cljs

```diff
@@ -36,9 +37,13 @@
                        mailserver-request-error?
                        :t/mailserver-request-error-status
 
+                       (and (= network-type "cellular")
```

Comment:
You've defined a function called `cellular?` Maybe we can re-use it  instead of keeping the hardcoded values?